### PR TITLE
i18n(es): update deploy guides

### PR DESCRIPTION
**Pull request by antonyfaris**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- Update with these changes:
  - [guides/deploy.md](https://github.com/withastro/docs/commits/main/src/pages/en/guides/deploy.md?since=2022-08-24T23:34:17.000Z)
  - [guides/deploy/cloudflare.md](https://github.com/withastro/docs/commits/main/src/pages/en/guides/deploy/cloudflare.md?since=2022-08-27T23:04:47.000Z)
  - [guides/deploy/deno.md](https://github.com/withastro/docs/commits/main/src/pages/en/guides/deploy/deno.md?since=2022-08-24T21:23:17.000Z)
  - [guides/deploy/github.md](https://github.com/withastro/docs/commits/main/src/pages/en/guides/deploy/github.md?since=2022-08-18T22:00:08.000Z)
  - [guides/deploy/netlify.md](https://github.com/withastro/docs/commits/main/src/pages/en/guides/deploy/netlify.md?since=2022-08-20T08:32:29.000Z)
  - [guides/deploy/vercel.md](https://github.com/withastro/docs/commits/main/src/pages/en/guides/deploy/vercel.md?since=2022-08-17T21:14:04.000Z)

### CODE REVIEW

**Comment by diegognt** [comment]
LGTM

File: src/pages/es/guides/deploy/cloudflare.md [hunk 1]
```diff
@@ -64,14 +64,21 @@ Entonces será posible actualizar el script preview en tu `package.json` por `"p
...
-Para habilitar un sitio con SSR y desplegarlo en Cloudflare Pages, necesitarás:
+Añade el adaptador de Cloudflare que activa SSR en tu proyecto de Astro con el siguiente comando `astro add` que se muestra debajo. Este instalará el adaptador y hará los cambios apropiados a tu archivo `astro.config.mjs` en un solo paso.
 
-1. Añadir el [adaptador `@astrojs/cloudflare`](https://github.com/withastro/astro/tree/main/packages/integrations/cloudflare#readme) al `package.json` de tu proyecto ejecutando:
+```bash
+npx astro add cloudflare
+```
+
+
+Si prefieres instalar el adaptador manualmente, sigue los siguientes dos pasos:
```
Comment (kevinzunigacuellar) [inline src/pages/es/guides/deploy/cloudflare.md:76]:
```suggestion
Si prefieres instalar el adaptador manualmente, sigue los siguientes pasos:

```

File: src/pages/es/guides/deploy/cloudflare.md [hunk 2]
```diff
@@ -64,14 +64,21 @@ Entonces será posible actualizar el script preview en tu `package.json` por `"p
...
+
+1. Añadir el adaptador `@astrojs/cloudflare` a las dependencias de tu proyecto usando tu manejador de paquetes preferido. Si estás usando npm o no estás seguro, ejecuta esto en la terminal:
```
Comment (kevinzunigacuellar) [inline src/pages/es/guides/deploy/cloudflare.md:78]:
```suggestion
1. Añadir el adaptador `@astrojs/cloudflare` a las dependencias de tu proyecto usando tu gestor de paquetes preferido. Si estás usando npm o no estás seguro, ejecuta esto en la terminal:

```

File: src/pages/es/guides/deploy/deno.md [hunk 3]
```diff
@@ -21,7 +21,16 @@ Tu proyecto de Astro puede ser desplegado en [Deno Deploy](https://deno.com/depl
...
+Añadir [el adaptador de Deno](/es/guides/integrations-guide/deno/) para activar SSR en tu proyecto de Astro con el siguiente comando `astro add` que se muestra debajo. Este instalará el adaptador y hará los cambios apropiados a tu archivo `astro.config.mjs` en un solo paso.
+
+
+```bash
+npx astro add deno
+```
+
+Si prefieres instalar el adaptador manualmente, sigue los siguientes dos pasos:
+
+1. Añadir [el adaptador `@astrojs/deno`](https://github.com/withastro/astro/tree/main/packages/integrations/deno) a las dependencias de tu proyecto usando tu manejador de paquetes preferido. Si estás usando npm o no estás seguro, ejecuta esto en la terminal:
```
Comment (kevinzunigacuellar) [inline src/pages/es/guides/deploy/deno.md:33]:
```suggestion
1. Añadir [el adaptador `@astrojs/deno`](https://github.com/withastro/astro/tree/main/packages/integrations/deno) a las dependencias de tu proyecto usando tu gestor de paquetes preferido. Si estás usando npm o no estás seguro, ejecuta esto en la terminal:
```

File: src/pages/es/guides/deploy/github.md [hunk 4]
```diff
@@ -14,16 +14,36 @@ Puedes desplegar un proyecto de Astro en GitHub pages usando [GitHub Actions](ht
...
+
+    :::note
+      No necesitas configuar el parametro `base` si:
+
+    - Tu repositorio se llama `<TU_NOMBRE_DE_USUARIO>.github.io`.
+    - Estás usando un dominio personalizado.
+    :::
+
+    :::caution
+        Si previamente no configuraste un valor para `base`, y acabas de configurarlo para desplegar en GitHub, ahora debes actualizar tus enlaces internos que ahora deben incluir `base`
```
Comment (kevinzunigacuellar) [inline src/pages/es/guides/deploy/github.md:37]:
```suggestion
        Si previamente no configuraste un valor para `base`, y acabas de configurarlo para desplegar en GitHub, recuerda actualizar tus enlaces internos para que incluyan la `base`
```

File: src/pages/es/guides/deploy/github.md [hunk 5]
```diff
@@ -65,18 +85,18 @@ Astro mantiene la action oficial `withastro/action` para desplegar tu proyecto c
     La [action oficial de Astro](https://github.com/withastro/action) busca un archivo lockfile para detectar tu gestor de paquetes (`npm`, `yarn` o `pnpm`). Debes tener el archivo `package-lock.json`, `yarn.lock` o `pnpm-lock.yaml`, generado automáticamente por tu gestor de paquetes, en tu repositorio.
     :::
 
-3. Confirma el nuevo workflow y envíalo a GitHub.
+3. En GitHub, ve a la pestaña **Settings** de tu repositorio y busca la sección **Pages**.
 
-4. En GitHub, ve a la pestaña **Settings** de tu repositorio y busca la sección **Pages**.
+4. Elige **GitHub Actions** como el **Source** de tu sitio y presiona **Save**.
```
Comment (kevinzunigacuellar) [inline src/pages/es/guides/deploy/github.md:90]:
```suggestion
4. Elige **GitHub Actions** como el **Source** de tu proyecto y presiona **Save**.  
```

File: src/pages/es/guides/deploy/github.md [hunk 6]
```diff
@@ -65,18 +85,18 @@ Astro mantiene la action oficial `withastro/action` para desplegar tu proyecto c
...
+3. En GitHub, ve a la pestaña **Settings** de tu repositorio y busca la sección **Pages**.
 
-4. En GitHub, ve a la pestaña **Settings** de tu repositorio y busca la sección **Pages**.
+4. Elige **GitHub Actions** como el **Source** de tu sitio y presiona **Save**.  
 
-5. Elige la rama `gh-pages` y la carpeta `"/" (raíz)` como **Source** de tu proyecto y presiona **Save**.
+5. Has *commit* del nuevo archivo workflow y has *push* a GitHub. 
 
-¡Tu proyecto está listo para ser desplegado! Cuando haya cambios en tu repositorio, GitHub Action los implementará automáticamente por ti.
+¡Tu sitio debería estar publicado! Cuando haya cambios en tu repositorio, GitHub Action los implementará automáticamente por ti.
```
Comment (kevinzunigacuellar) [inline src/pages/es/guides/deploy/github.md:94]:
```suggestion
¡Tu proyecto debería estar publicado! Cuando haya cambios en tu repositorio, GitHub Action los implementará automáticamente por ti.
```

File: src/pages/es/guides/deploy/netlify.md [hunk 7]
```diff
@@ -20,31 +20,37 @@ Tu proyecto de Astro es un sitio estático por defecto. No necesitas ninguna con
 
 Para habilitar SSR en tu proyecto de Astro y hacer un despliegue en Netlify:
 
-1. Instala [el adaptador de Netlify](https://github.com/withastro/astro/tree/main/packages/integrations/netlify) en las dependencias de tu proyecto.
+Añadir [el adaptador de Netlify](/es/guides/integrations-guide/netlify/) en tu proyecto de Astro con el siguiente comando `astro add` que se muestra debajo. Este instalará el adaptador y hará los cambios apropiados a tu archivo `astro.config.mjs` en un solo paso.
```
Comment (kevinzunigacuellar) [inline src/pages/es/guides/deploy/netlify.md:23]:
```suggestion
Añade [el adaptador de Netlify](/es/guides/integrations-guide/netlify/) a tu proyecto de Astro con el siguiente comando `astro add` que se muestra debajo. Este instalará el adaptador y hará los cambios apropiados a tu archivo `astro.config.mjs` en un solo paso.
```

File: src/pages/es/guides/deploy/netlify.md [hunk 8]
```diff
@@ -20,31 +20,37 @@ Tu proyecto de Astro es un sitio estático por defecto. No necesitas ninguna con
...
+
+```bash
+npx astro add netlify
+```
+
+Si prefieres instalar el adaptador manualmente, sigue los siguientes dos pasos:
+
+1. Añade [el adaptador de `@astrojs/netlify`](https://github.com/withastro/astro/tree/main/packages/integrations/netlify) a las dependencias de tu proyecto usando tu manejador de paquetes preferido. Si estás usando npm o no estás seguro, ejecuta esto en la terminal:
```
Comment (kevinzunigacuellar) [inline src/pages/es/guides/deploy/netlify.md:31]:
```suggestion
1. Añade [el adaptador de `@astrojs/netlify`](https://github.com/withastro/astro/tree/main/packages/integrations/netlify) a las dependencias de tu proyecto usando tu gestor de paquetes preferido. Si estás usando npm o no estás seguro, ejecuta esto en la terminal:
```

File: src/pages/es/guides/deploy/vercel.md [hunk 9]
```diff
@@ -16,28 +16,45 @@ Tu proyecto de Astro puede ser desplegado en Vercel como un sitio estático, o c
...
+}
+```
+:::
 
 ### Adaptador para SSR
 
 Para habilitar SSR en tu proyecto de Astro y desplegar en Vercel:
 
-1. Instala [el adaptador de Vercel](https://github.com/withastro/astro/tree/main/packages/integrations/vercel) en las dependencias de tu proyecto.
+Añadir [el adaptador de Vercel](/es/guides/integrations-guide/vercel/) en tu proyecto de Astro con el siguiente comando `astro add` que se muestra debajo. Este instalará el adaptador y hará los cambios apropiados a tu archivo `astro.config.mjs` en un solo paso.
```
Comment (kevinzunigacuellar) [inline src/pages/es/guides/deploy/vercel.md:35]:
```suggestion
Añade [el adaptador de Vercel](/es/guides/integrations-guide/vercel/) en tu proyecto de Astro con el siguiente comando `astro add` que se muestra debajo. Este instalará el adaptador y hará los cambios apropiados a tu archivo `astro.config.mjs` en un solo paso.

```

File: src/pages/es/guides/deploy/vercel.md [hunk 10]
```diff
@@ -16,28 +16,45 @@ Tu proyecto de Astro puede ser desplegado en Vercel como un sitio estático, o c
...
+
+```bash
+npx astro add vercel
+```
+
+Si prefieres instalar el adaptador manualmente, sigue los siguientes dos pasos:
+
+1. Añade [el adaptador de `@astrojs/vercel`](/es/guides/integrations-guide/vercel/) a las dependencias de tu proyecto usando tu manejador de paquetes preferido. Si estás usando npm o no estás seguro, ejecuta esto en la terminal:
```
Comment (kevinzunigacuellar) [inline src/pages/es/guides/deploy/vercel.md:43]:
```suggestion
1. Añade [el adaptador de `@astrojs/vercel`](/es/guides/integrations-guide/vercel/) a las dependencias de tu proyecto usando tu gestor de paquetes preferido. Si estás usando npm o no estás seguro, ejecuta esto en la terminal:

```

**Review by kevinzunigacuellar — COMMENTED** [review]
a few things to keep vocabulary consistent across the esp docs

**Review by kevinzunigacuellar — APPROVED** [review]
good to go 🎉

**Pull request merged**